fabtests/scripts/runfabtests.sh

```diff
@@ -74,12 +74,19 @@ declare -i pass_count=0
 declare -i fail_count=0
 declare -i total_failures=0
 
+python=$(which python3 2>/dev/null) || python=$(which python2 2>/dev/null)
+
```

Comment:
IMO, if neither python2 nor python3 is available, we should exist and ask user to install python.

Reply:
We do exit, see line 79-82, are you asking for a different message?

Reply:
Sorry, I missed that.